.editorconfig

```diff
@@ -0,0 +1,15 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_size = 2
+quote_type = single
+indent_style = space
+max_line_length = 80
+insert_final_newline = false
```

Comment:
I am sorry for the suggestion, but please let `insert_final_newline` be true. 

```suggestion
```

Reply:
Accepted.